File: PST/FUEL Source Code/FUEL/FS/Math.cs
// Decompiled with JetBrains decompiler
// Type: FUEL.FS.Math
// Assembly: FUEL, Version=2.4.8.0, Culture=neutral, PublicKeyToken=null
// MVID: A3D630D0-0620-425D-8CFB-B67B2AC75507
// Assembly location: C:\hp\FUEL\FUEL.dll

using System.Diagnostics;

namespace FUEL.FS
{
  public class Math
  {
    [DebuggerNonUserCode]
    public Math()
    {
    }

    public static bool IsCloseTo(double PrimaryVal, double SecondaryVal)
    {
      double Threshold = PrimaryVal * 0.1;
      return Math.IsCloseTo(PrimaryVal, SecondaryVal, Threshold);
    }

    public static bool IsCloseTo(double PrimaryVal, double SecondaryVal, double Threshold) => System.Math.Max(PrimaryVal, SecondaryVal) - System.Math.Min(PrimaryVal, SecondaryVal) <= Threshold;
  }
}

File: PST/FUEL/src/FUEL/FS/Math.vb
Imports System

Namespace FUEL.FS
    Public Class Math
        ' Methods
        Public Shared Function IsCloseTo(ByVal PrimaryVal As Double, ByVal SecondaryVal As Double) As Boolean
            Dim threshold As Double = (PrimaryVal * 0.1)
            Return Math.IsCloseTo(PrimaryVal, SecondaryVal, threshold)
        End Function

        Public Shared Function IsCloseTo(ByVal PrimaryVal As Double, ByVal SecondaryVal As Double, ByVal Threshold As Double) As Boolean
            Return ((Math.Max(PrimaryVal, SecondaryVal) - Math.Min(PrimaryVal, SecondaryVal)) <= Threshold)
        End Function

    End Class
End Namespace

